web/src/main/java/org/akaza/openclinica/service/ParticipateServiceImpl.java

```diff
@@ -391,7 +391,7 @@ public boolean mayProceed(String studyOid) throws Exception {
         StudyParameterValueBean pStatus = spvdao.findByHandleAndStudy(pStudy.getId(), "participantPortal");
         String participateStatus = pStatus.getValue().toString();
 
-        if( participateStatus.equalsIgnoreCase("enabled") && study.getStatus().isAvailable()){
+        if( participateStatus.equals("enabled") && study.getStatus().isAvailable()){
```

Comment:
Instead of hardcoding "enabled", could you use ModuleProcessor.ModuleStatus.ENABLED.getValue() here as well?